### PR TITLE
Add connection documention to firmata

### PR DESCRIPTION
**Pull request by zankavrogin**

### CODE REVIEW

File: platforms/firmata/README.md [hunk 1]
```diff
@@ -48,6 +48,63 @@ func main() {
...
+
+First plug the Arduino into your computer via the USB/serial port.
+A dialog box will appear telling you that a new network interface has been detected.
+Click "Network Preferences...", and when it opens, simply click "Apply".
+
+Once plugged in, use [Gort](http://gort.io)'s `gort scan serial` command to find out your connection info and serial port address:
+
+```
+$ gort scan serial
+```
```
Comment (stewart) [inline platforms/firmata/README.md]: these can just be indented 4 spaces, rather than surrounding as a fenced code block

**Comment by zankavrogin** [comment]
@stewart The way the import script is set up, it needs the fenced blocked of code to convert it to be Haml compatible. [Here](https://github.com/hybridgroup/gobot-site/blob/master/bin/import_platforms#L46) is the code.

**Comment by stewart** [comment]
Oh, that's right. (╯°□°）╯︵ ┻━┻

**Comment by zankavrogin** [comment]
I updated the code with the Windows connection documentation.

**Comment by deadprogram** [comment]
:book:

**Pull request merged**